src/core/utils/types/material-type.ts

```diff
@@ -1,15 +1,15 @@
 const enum MaterialType {
```

Comment:
Where does these "translations" come from? Like why is `game = "playstation 5"`

Reply:
These are term names for the material types provided by Tue.